app/assets/stylesheets/new_styles/_forms.scss

```diff
@@ -103,3 +103,7 @@ form.block-form {
     ::placeholder { text-transform: uppercase; }
   }
 }
+
+.form_errors {
+  color : #b94a48;
```

Comment:
Please use a color variable here. You could try `$red`. If that is not ok for you please add this color as a new variable in `colors.scss`.